api/v1beta1/tempest_types.go

```diff
@@ -351,6 +351,11 @@ type TempestSpec struct {
 	// spawned by the test operator.
 	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
 
+	// +kubebuilder:validation:Optional
+        // +kubebuilder:default=false
+        // Activate debug mode
```

Comment:
Can the explanation of this parameter be a little bit expanded? Maybe something like this (?):

```
Activate debug mode. When debug mode is activated any error encountered inside 
the test-pod causes that the pod will be kept alive indefinitely (stuck in "Running" phase) 
or until the corresponding Tempest CR is deleted This allows to debug any potential troubles
with `oc rsh`.
```

Maybe you'll find a better way how to describe this feature:).

Reply:
Also do not forget to run `make generate && make manifests` after you update the string here:)

Reply:
Very nicely written, I have changed the description and also ran the `make generate && make manifests` commands.

Reply:
Thank you!:)